src/Components/Publishing/Article.tsx

```diff
@@ -94,62 +65,58 @@ class Article extends React.Component<ArticleProps, ArticleState> {
+    const { article } = this.state
+    const relatedArticlePanel = relatedArticlesForPanel
+      ? <RelatedArticlesPanel label={"Related Stories"} articles={relatedArticlesForPanel} />
+      : false
+    const relatedArticleCanvas = relatedArticlesForCanvas
+      ? <RelatedArticlesCanvas articles={relatedArticlesForCanvas} vertical={article.vertical} />
+      : false
+    const emailSignup = this.props.emailSignupUrl ? <EmailSignup signupUrl={this.props.emailSignupUrl} /> : false
+    return (
+      <div>
+        <ReadMoreWrapper isTruncated={this.state.isTruncated} hideButton={this.removeTruncation}>
+          <Header article={article} />
+          <StandardLayout>
             <Sections article={article} />
-          </FeatureLayout>
-          <FullscreenViewer
-            onClose={this.closeViewer}
-            show={this.state.viewerIsOpen}
-            slideIndex={this.state.slideIndex}
-            images={this.state.fullscreenImages}
-          />
-        </ArticleContainer>
-      )
-    } else {
-      const relatedArticlePanel = relatedArticlesForPanel ? (
-        <RelatedArticlesPanel label={"Related Stories"} articles={relatedArticlesForPanel} />
-      ) : (
-        false
-      )
-      const relatedArticleCanvas = relatedArticlesForCanvas ? (
-        <RelatedArticlesCanvas articles={relatedArticlesForCanvas} vertical={article.vertical} />
-      ) : (
-        false
-      )
-      const emailSignup = this.props.emailSignupUrl ? <EmailSignup signupUrl={this.props.emailSignupUrl} /> : false
-      return (
-        <ArticleContainer marginTop={marginTop}>
-          <ReadMoreWrapper isTruncated={this.state.isTruncated} hideButton={this.removeTruncation}>
-            <Header article={article} />
-            <StandardLayout>
-              <Sections article={article} />
-              <Sidebar>
-                {emailSignup}
-                {relatedArticlePanel}
-              </Sidebar>
-            </StandardLayout>
-            {relatedArticleCanvas}
-          </ReadMoreWrapper>
-          {this.state.isTruncated ? <ReadMore onClick={this.removeTruncation} /> : false}
-          <FullscreenViewer
-            onClose={this.closeViewer}
-            show={this.state.viewerIsOpen}
-            slideIndex={this.state.slideIndex}
-            images={this.state.fullscreenImages}
-          />
-        </ArticleContainer>
-      )
-    }
+            <Sidebar>
+              {emailSignup}
+              {relatedArticlePanel}
+            </Sidebar>
+          </StandardLayout>
+          {relatedArticleCanvas}
+        </ReadMoreWrapper>
+        {this.state.isTruncated ? <ReadMore onClick={this.removeTruncation} /> : false}
+      </div>
+    )
+  }
+
+  render() {
+    const { marginTop } = this.props
+    const { article } = this.state
+    return (
+      <ArticleContainer marginTop={marginTop}>
+        {article.layout === "feature" ? this.renderFeatureArticle() : this.renderStandardArticle()}
```

Comment:
Reminds me of this comment by @damassi: https://github.com/artsy/reaction/pull/263#discussion_r143336391

I never looked into why or when this can cause issues though. Any experience with that?

Reply:
In JSX this is common since it returns a value (renderable JSX)! But here's an illustration of a naked ternary (non-assignment) that's dangerous: 

```javascript
const isOpen = true
isOpen ? foo() : orSomeTrutyValue()
additionalCodeToExecute()
```
When run, `additionalCodeToExecute` will move up to the `else` conditional since there's no semi denoting a break. Most linters guard against this type of thing.

Reply:
Ah, I see what Chris is saying there—in that context you linked there are rare circumstances where using it in that imperitive way (e.g. not assigned to a variable or used within an expression as is the case here) combined with JS's automatic semicolon insertion (b/c we don't use mandatory semis) could cause an issue.

That said, it should be totally fine to use the terery in JSX expressions like this.

Reply:
OoOo ic. Thanks for the explanation!